lib/widgets/content.dart

```diff
@@ -1030,7 +1037,7 @@ class _InlineContentBuilder {
         _pushRecognizer(recognizer);
         final result = _buildNodes(node.nodes,
           // Web has the same color in light and dark mode.
```

Comment:
Let's remove this comment; it would be easy for it to become wrong if the light/dark variants start to be different. Also you've already left a comment with the same meaning on the dark variant.